src/habitat.cr

```diff
@@ -2,14 +2,16 @@ require "./habitat/*"
 
 class Habitat
   class MissingSettingError < Exception
-    def initialize(type, setting_name)
+    def initialize(type, setting_name, example)
+      example = "some_value" if example == Nil
```

Comment:
You could remove this by setting the example in the macro https://github.com/luckyframework/habitat/pull/27/files?utf8=✓&diff=split&w=1#diff-a82a4bbf3e7cdad8547a42db8e22426aR75